Source/NetworkError.swift

```diff
@@ -36,7 +36,7 @@ public enum NetworkError: Error {
     /// Error on the server (HTTP Error 500...511)
     case serverError(response: HTTPURLResponse?, data: Data?)
     /// Parsing the body into expected type failed.
-    case serializationError(error: Error, data: Data?)
+    case serializationError(error: Error, response: HTTPURLResponse?, data: Data?)
```

Comment:
Das hier ist ein Breaking Change und müsste dann in die geplante v4.0. Lass dazu gerne mal sprechen.

HTTPURLResponse? sollte eigentlich HTTPURLResponse sein, da bei einem  serializationError meiner Meinung nach immer eine Response mitkommen müsste?